test/Bugs/randombug.baseline

```diff
@@ -63,10 +63,10 @@ Test case 30
 Passed
 Test case 31
 Error
-	at func2 (randombug.js:287:9)

```

Comment:
I think we could change the implementation of TrimStackTracePath or add an alternate function which additionally trims line numbers. But I'd say that's out of scope of this change. I don't want to consider the intent behind testing (or not) the line numbers of error stack traces.

---
In reply to: [107286850](https://github.com/Microsoft/ChakraCore/pull/2726#discussion_r107286850) [](ancestors = 107286850)